content/en/docs/reference/tfjob/v1beta2/common.md

```diff
@@ -59,9 +59,7 @@ JobConditionType
 <td>
 <code>status</code></br>
 <em>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#conditionstatus-v1-core">
-Kubernetes core/v1.ConditionStatus
-</a>
+string
```

Comment:
@Arhell is this `string` on purpose?

Reply:
@RFMVasconcelos
you are right, this is what I found, but the link is not working unfortunately here too
`status`
https://aws.github.io/aws-app-mesh-controller-for-k8s/reference/api_spec/#appmesh.k8s.aws/v1beta2.GatewayRouteCondition